sequelize/data/user-role.json

```diff
@@ -1,6 +1,73 @@
 [
     {
         "role": "admin",
```

Comment:
Is there an easier way to just let an admin do everything here? Just curious

Reply:
Unfortunately from what I've seen with Casbin you cannot wildcard the method, it has to be explicit.